src/rust/inetstack/protocols/udp/mod.rs

```diff
@@ -9,7 +9,7 @@
 
 mod datagram;
 mod futures;
-mod peer;
+pub mod peer;
```

Comment:
Why do we need to expose this?

Reply:
Yes because the UdpQueue struct is defined there.